sdk/identity/identity/README.md

```diff
@@ -79,31 +99,23 @@ const client = new KeyClient(vaultUrl, credential);
 const getResult = await client.getKey("MyKeyName");
 ```
 
-### Authenticating as a service principal
+### Specifying a user assigned managed identity with the `DefaultAzureCredential`
 
-```javascript
-// Using a client secret
-const { ClientSecretCredential } = require("@azure/identity");
-const credential = new ClientSecretCredential(tenantId, clientId, clientSecret);
-
-// Using a PEM-encoded certificate with a private key, not password protected
-const { ClientCertificateCredential } = require("@azure/identity");
-const credential = new ClientCertificateCredential(
-  tenantId,
-  clientId,
-  "/app/certs/certificate.pem"
-);
-
-// Using environment variables (see "Environment variables" above for variable names)
-const { EnvironmentCredential } = require("@azure/identity");
-const credential = new EnvironmentCredential();
-```
+Many Azure hosts allow the assignment of a user assigned managed identity. This example demonstrates configuring the `DefaultAzureCredential` to authenticate a user assigned identity when deployed to an azure host. It then authenticates a `KeyClient` from the [@azure/keyvault-keys](https://www.npmjs.com/package/@azure/keyvault-keys) client library with credential.
 
-### Using the `AuthorizationCodeCredential`
+```ts
+const { KeyClient } = require("@azure/keyvault-keys");
+const { DefaultAzureCredential } = require("@azure/identity");
 
-The `AuthorizationCodeCredential` takes more up-front work to use than the other credential types at this time. A full sample demonstrating how to use this credential can be found in [`samples/authorizationCodeSample.ts`](samples/authorizationCodeSample.ts).
+// when deployed to an azure host the default azure credential will authenticate the specified user assigned managed identity
+var credential = new DefaultAzureCredential({ managedIdentityClientId: userAssignedClientId });
```

Comment:
This section was requested by @schaabs , but it does require this new PR to be reviewed and merged: ptionally pass in a user assigned client ID for the ManagedIdentityCredential